### DISCUSSION TITLE
Runaway Resistance - Dopamine Hit

### DISCUSSION THREAD

**Discussion by Zvirani786** [cagnulein/qdomyos-zwift#1217] [Ideas]
Hello,
I was looking to get all avelable options to customize the code. I was thinkin of code like this:

Duration, zoneHR, HR loop, target cadence, max useable resistance

Especially the max useable resistance I think is important because the cycle's auto resistance goes into runaway mode when u get tired and cannot keep up but want to also take a breather and not quit.

Also if there can b a color and audio indicators to show when you are on target. If possible I would like to set a target cadence, HR and the numbers change color to green and a audio sound when they are all green just to reinforce that you are doing well. This will make it a kind of game, a reachable goal and give you a nice dopamine hit to have you work harder.
Thank You

**Comment by cagnulein** [comment]
@Zvirani786 hi! are you talking about train program with xml format?
if so yes I can add maxresistance as there is already maxspeed for treadmill.

for the audio there is already a ticket #1187 could you comment this one?

for the color i guess the target cadence is already doing this, isn't it?

let me know!

**Reply by Zvirani786** [reply]
Hello,
Yes it's for the xml format. And I did find the post with all the options to create a custom workout. I also say the post of the sound that you are trying to create aswell. Great job!
Thank

**Reply by cagnulein** [reply]
ok so for the max resistance, please fill a new ticket and i will add it!
